json_schema/core/file/file_core.json

```
@@ -48,7 +48,47 @@
             "description": "MD5 checksum of the file.",
             "type": "string",
             "user_friendly": "Checksum",
-            "example": "e09a986c2e630130b1849d4bf9a94c06"
-	    }
+            "example": "e09a986c2e630130b1849d4bf9a94c06",
+            "pattern": "^[a-f0-9]{32}$"
+	    },
+        "content_type": {
+            "description": "An appropriate MIME type for this file.",
+            "type": "string",
+            "user_friendly": "Content type",
+            "example": "application/gzip"
+        },
+        "size": {
+            "description": "The size of the file in bytes.",
+            "type": "number",
+            "minimum": 0,
+            "user_friendly": "Size",
+            "example": 2147483648
+        },
+        "sha256": {
+            "description": "The sha256 hash of the file.",
+            "type": "string",
+            "pattern": "^[A-Fa-f0-9]{64}",
```

```suggestion
            "pattern": "^[a-f0-9]{64}",
```

Since so many people incorrectly perform a case-sensitive comparison, we shouldn't allow an ambiguous hexadecimal representation. Ambiguity is the mother of doom.